bin/run_multiple_panels.py

```diff
@@ -0,0 +1,507 @@
+            clock = dwelling.current_time
+            if shed_ev(house_status, size) is not None:
+                N += 1
+                t_delay.append(dwelling.current_time)
+        elif dwelling.current_time - clock < pd.Timedelta(15, "m"):
+            if clock == dwelling.start_time: # no EV load has been shedded
+                control_signal = None
+            else:
+                control_signal = {'EV': {'P Setpoint': shed_ev(house_status, size)}}
+                if shed_ev(house_status, size) is not None:
+                    N += 1
+                    t_delay.append(dwelling.current_time)
+        else:
+            # Keep previous control signal (same as forward fill)
+            control_signal = None
+
+    df = pd.DataFrame(t_delay, columns=['Timestamp'])
+    df.to_csv(os.path.join(output_path, 'EV_metrics.csv'), index=False)
+    
+    dwelling.finalize()
+
+    
+def compile_results(main_folder):
+    # Sample script to compile results from multiple OCHRE runs
+    # assumes each run is in a different folder, and all simulation names are 'ochre'
+    # dirs_to_include = int(dirs_to_include)
+
+    # set up
+    main_folder = os.path.abspath(main_folder)
+    output_folder = os.path.join(main_folder, 'compiled')
+    os.makedirs(output_folder, exist_ok=True)
+    my_print('Compiling OCHRE results for:', main_folder)
+
+    # find all building folders
+    required_files = ['ochre.parquet', 'ochre_metrics.csv', 'ochre_complete']
+    run_folders = Analysis.find_subfolders(main_folder, required_files)
+    n = len(run_folders)
+    my_print(f'Found {n} folders with completed OCHRE simulations')
+
+    # combine input json files
+    df = Analysis.combine_json_files(path=main_folder)
+    df.to_csv(os.path.join(output_folder, 'all_ochre_inputs.csv'))
+
+    # combine metrics files
+    df = Analysis.combine_metrics_files(path=main_folder)
+    df.to_csv(os.path.join(output_folder, 'all_ochre_metrics.csv'))
+
+    # combine total electricity consumption for each house
+    results_files = [os.path.join(f, 'ochre.parquet') for f in run_folders]
+    df = Analysis.combine_time_series_column(results_files, 'Total Electric Power (kW)')
+    df.to_parquet(os.path.join(output_folder, 'all_ochre_powers.parquet'))
+
+    # aggregate outputs for each house (sum or average only)
+    _, _, df_single = Analysis.load_ochre(run_folders[0], 'ochre', load_main=False)
+    agg_func = {col: Analysis.get_agg_func(col, 'House') for col in df_single.columns}
+    df = Analysis.combine_time_series_files(results_files, agg_func)
+    df.to_parquet(os.path.join(output_folder, 'all_ochre_results.parquet'))
+
+    my_print('All compiling complete')
+
+
+def my_print(*args):
+    # prints with date and other info
+    now = dt.datetime.now()
+    print(now, *args)
+
+
+if __name__ == "__main__":
+    
+    # read the spreadsheet containing all scenarios
+    scenarios = pd.read_csv(os.path.join(os.getcwd(), 'bin', 'control_scenarios_processed.csv'))
```

Comment:
I would remove this and replace it with 3 scenarios that can all be run without any additional data:
 - Circuit sharing: no dryer, maybe EV and WH?
 - Circuit pausing
 - EV shed

Maybe you can collect the data from each run and use the compile_results function after? I don't think we have any working examples with some of those functions, so that would be nice to keep.

Reply:
I would at least need an hpxml file and a schedules file in order to run this script. Where should I be keeping those files?

Reply:
In here: https://github.com/NREL/OCHRE/tree/main/ochre/defaults/Input%20Files

We're going to be updating the existing files to a ResStock 2024 model with an all electric home. See here for what will probably get pushed:
https://github.com/NREL/OCHRE/tree/153-update-examples/ochre/defaults/Input%20Files

Reply:
I added required input files into the default input path.